### ISSUE TITLE
Booking Overview - UI

### ISSUE THREAD

**Issue by Ardianpalloshi** [amosproj/amos2022ws05-shared-desk-mgmt#21]
## User story
1. As a user
2. I want / need to see my bookings
3. So that I remember when to go to the office and can delete or edit the bookings if necessary

## Acceptance criteria
* The UI page should be identifyable as the booking overview page (e.g through header)
* The UI page should display all bookings with the desk, date and time details for that user ordered by date/time
* The page should provide access to some edit and delete functionality for the bookings

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by qy21gafy** [comment]
The page should is not the overview on the overview page, but a new section available through the top bar. The overview page displays the 10 (or so) recent bookings, this page displays all bookings in the future with a visible edit and delete button next to each booking.
